package/src/components/AutoCompleteInput/AutoCompleteInput.tsx

```diff
@@ -108,16 +108,18 @@ const AutoCompleteInputWithContext = (props: AutoCompleteInputPropsWithContext)
   }, [command, cooldownActive, t]);
 
   const handleContentSizeChange = useCallback(
-    ({
-      nativeEvent: { contentSize },
-    }: NativeSyntheticEvent<TextInputContentSizeChangeEventData>) => {
+    ({ nativeEvent: { contentSize } }: TextInputContentSizeChangeEvent) => {
       setTextHeight(contentSize.height);
     },
     [],
   );
 
+  if (TextInputComponent) {
+    return <TextInputComponent {...rest} />;
```

Comment:
Please make sure the passed props are the same here, so that the overidden component can be used directly and integrators can decide what to keep and what to remove. Otherwise, stuff like `autoFocus`, `multiline` and what not will break (and so will our UI).

Reply:
Done